internal/federationdomain/endpoints/auth/auth_handler.go

```diff
@@ -34,6 +35,21 @@ const (
 	promptParamNone = "none"
 )
 
+//nolint:gochecknoglobals // please treat this as a readonly const, do not mutate
+var paramsSafeToLog = sets.New[string](
```

Comment:
we could also log the error paramers from OAuth2 callbacks

Reply:
Note that's on the `/callback` endpoint, not the `/authorize` handler

Reply:
Done!